pages/articles.tsx

```diff
@@ -28,6 +33,26 @@ export default function Articles({ tags, posts }) {
             ))
             .reduce((prev, curr) => [prev, ", ", curr])}
         </p>
+        <div className="stork">
+          <div className="stork-wrapper">
+            <input placeholder="Search all posts.." data-stork="posts" className="stork-input" />
+            <div data-stork="posts-output" className="stork-output"></div>
+          </div>
+        </div>
+        <Script
+          src="https://files.stork-search.net/stork.js"
+          onLoad={() => {
+            // @ts-ignore
```

Comment:
Curious - what does TS complain about here?

Reply:
`Property 'stork' does not exist on type 'Window & typeof globalThis'.ts(2339)`

It didn't like my attempted hack of:

```ts
window as Window & { stork: { register: (a, b) => {}} }
```

I think the solution is a stork library + types (and not loading it via `<script>`).